arviz/plots/pairplot.py

```diff
@@ -34,6 +35,8 @@ def plot_pair(
     data : obj
         Any object that can be converted to an az.InferenceData object
         Refer to documentation of az.convert_to_dataset for details
+    group : {"prior", "posterior"}, optional
```

Comment:
In this case I would not restrict group to be either prior or posterior, in some cases it may be useful to plot other groups such as the posterior predictive.

Reply:
Okay, will do.